### ISSUE TITLE
General media widget in stackable widget 

### ISSUE THREAD

**Issue by flnEGknEF** [8bitPit/Niagara-Issues#2271]
![Screenshot_20230507-103251](https://user-images.githubusercontent.com/10791832/236700777-e56d39ee-7d5b-49b0-b4d4-48f7a271322c.png)
I'd like to see a "general" media widget within the stackable widget part of the UI so that any media being played from any source could me played/paused within the stackable widget. This would keep users from having to fill all 4 stackable widget slots with 4 different widget sources (Spotify, YouTube, etc)

**Comment by peit-uiberry** [comment]
There are music widgets available on the playstore that accomplish exactly that. They're slightly different from the default launcher one but accomplish the same thing. KWGT is also an option. Remember to disable the default music widget

**Comment by flnEGknEF** [comment]
Yeah I am using one but I would prefer it be included in Niagara as it would be built in the launcher the same way the standard media player widget is and would more likely work with all platforms no matter which app was launched. An all in one

**Comment by Amorphous404** [comment]
Hi @flnEGknEF, thank you for your suggestion. We may consider incorporating an independent media widget in the future. However, it is not currently among our top priorities.

Until it's available, as recommended by @peit-uiberr, you can make use of third-party applications such as [Audio Widgets](https://play.google.com/store/apps/details?id=com.widgets.music) and [Android 12 Music Widget](https://play.google.com/store/apps/details?id=com.lstapps.musicwidgetandroid12) to achieve similar functionality and combine them with your other widgets.
